Translated/Story/08-09 Finding Bruder/msq_011501.csv

```diff
@@ -0,0 +1,2 @@
+msq_011501_0010#0,"""We've got to find something that'll serve as the headstone first. When we get of this cave, let's check towards the forest."""
```

Comment:
"When we get **out of** this cave."

And I don't like "check towards". "check around" or "head towards", maybe?

Reply:
we never even enter a cave during this quest so i honestly dont know where this file is even used. fixed anyways lol